app/auth/FogotPassword/page.jsx

```diff
@@ -0,0 +1,90 @@
+"use client";
+import { Box, Button, TextField } from "@mui/material";
+import Layout from "../layput";
+import imgageicon from '../images/forgotIcon.png'
```

Comment:
I think we should save the image as svg.
It will make the website render image faster

Reply:
ok, confirm that